[2024-10-16T09:55:50.480+0000] {taskinstance.py:1087} INFO - Dependencies all met for <TaskInstance: exchange_rate_usd_kzt_dag.insert_rate scheduled__2024-06-05T00:00:00+00:00 [queued]>
[2024-10-16T09:55:50.500+0000] {taskinstance.py:1087} INFO - Dependencies all met for <TaskInstance: exchange_rate_usd_kzt_dag.insert_rate scheduled__2024-06-05T00:00:00+00:00 [queued]>
[2024-10-16T09:55:50.501+0000] {taskinstance.py:1283} INFO - 
--------------------------------------------------------------------------------
[2024-10-16T09:55:50.502+0000] {taskinstance.py:1284} INFO - Starting attempt 1 of 1
[2024-10-16T09:55:50.503+0000] {taskinstance.py:1285} INFO - 
--------------------------------------------------------------------------------
[2024-10-16T09:55:50.528+0000] {taskinstance.py:1304} INFO - Executing <Task(PostgresOperator): insert_rate> on 2024-06-05 00:00:00+00:00
[2024-10-16T09:55:50.536+0000] {standard_task_runner.py:55} INFO - Started process 8876 to run task
[2024-10-16T09:55:50.543+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'exchange_rate_usd_kzt_dag', 'insert_rate', 'scheduled__2024-06-05T00:00:00+00:00', '--job-id', '1803', '--raw', '--subdir', 'DAGS_FOLDER/exchange_rates/__init__.py', '--cfg-path', '/tmp/tmp_qfoac4f']
[2024-10-16T09:55:50.545+0000] {standard_task_runner.py:83} INFO - Job 1803: Subtask insert_rate
[2024-10-16T09:55:50.703+0000] {task_command.py:389} INFO - Running <TaskInstance: exchange_rate_usd_kzt_dag.insert_rate scheduled__2024-06-05T00:00:00+00:00 [running]> on host 63ee4fa64e74
[2024-10-16T09:55:50.775+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.7/site-packages/***/utils/context.py:205: AirflowContextDeprecationWarning: Accessing 'execution_date' from the template is deprecated and will be removed in a future version. Please use 'data_interval_start' or 'logical_date' instead.
  warnings.warn(_create_deprecation_warning(key, self._deprecation_replacements[key]))

[2024-10-16T09:55:50.850+0000] {taskinstance.py:1513} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=exchange_rate_usd_kzt_dag
AIRFLOW_CTX_TASK_ID=insert_rate
AIRFLOW_CTX_EXECUTION_DATE=2024-06-05T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2024-06-05T00:00:00+00:00
[2024-10-16T09:55:50.852+0000] {sql.py:253} INFO - Executing: INSERT INTO
    currency_exchange_rates
VALUES ('USD', 'KZT', 447.78461655, '2024-06-05')
ON CONFLICT (base, currency, date) DO
    UPDATE
        SET rate = excluded.rate;
[2024-10-16T09:55:50.863+0000] {base.py:73} INFO - Using connection ID 'postgres_default' for task execution.
[2024-10-16T09:55:51.151+0000] {base.py:73} INFO - Using connection ID 'postgres_default' for task execution.
[2024-10-16T09:55:51.161+0000] {sql.py:364} INFO - Running statement: INSERT INTO
    currency_exchange_rates
VALUES ('USD', 'KZT', 447.78461655, '2024-06-05')
ON CONFLICT (base, currency, date) DO
    UPDATE
        SET rate = excluded.rate;, parameters: None
[2024-10-16T09:55:51.164+0000] {sql.py:373} INFO - Rows affected: 1
[2024-10-16T09:55:51.178+0000] {taskinstance.py:1327} INFO - Marking task as SUCCESS. dag_id=exchange_rate_usd_kzt_dag, task_id=insert_rate, execution_date=20240605T000000, start_date=20241016T095550, end_date=20241016T095551
[2024-10-16T09:55:51.215+0000] {local_task_job.py:159} INFO - Task exited with return code 0
[2024-10-16T09:55:51.244+0000] {taskinstance.py:2582} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-10-16T10:05:50.563+0000] {taskinstance.py:1087} INFO - Dependencies all met for <TaskInstance: exchange_rate_usd_kzt_dag.insert_rate scheduled__2024-06-05T00:00:00+00:00 [queued]>
[2024-10-16T10:05:50.581+0000] {taskinstance.py:1087} INFO - Dependencies all met for <TaskInstance: exchange_rate_usd_kzt_dag.insert_rate scheduled__2024-06-05T00:00:00+00:00 [queued]>
[2024-10-16T10:05:50.583+0000] {taskinstance.py:1283} INFO - 
--------------------------------------------------------------------------------
[2024-10-16T10:05:50.584+0000] {taskinstance.py:1284} INFO - Starting attempt 1 of 1
[2024-10-16T10:05:50.585+0000] {taskinstance.py:1285} INFO - 
--------------------------------------------------------------------------------
[2024-10-16T10:05:50.611+0000] {taskinstance.py:1304} INFO - Executing <Task(PostgresOperator): insert_rate> on 2024-06-05 00:00:00+00:00
[2024-10-16T10:05:50.617+0000] {standard_task_runner.py:55} INFO - Started process 11416 to run task
[2024-10-16T10:05:50.623+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'exchange_rate_usd_kzt_dag', 'insert_rate', 'scheduled__2024-06-05T00:00:00+00:00', '--job-id', '2521', '--raw', '--subdir', 'DAGS_FOLDER/exchange_rates/__init__.py', '--cfg-path', '/tmp/tmpzffund7a']
[2024-10-16T10:05:50.624+0000] {standard_task_runner.py:83} INFO - Job 2521: Subtask insert_rate
[2024-10-16T10:05:50.714+0000] {task_command.py:389} INFO - Running <TaskInstance: exchange_rate_usd_kzt_dag.insert_rate scheduled__2024-06-05T00:00:00+00:00 [running]> on host 63ee4fa64e74
[2024-10-16T10:05:50.770+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.7/site-packages/***/utils/context.py:205: AirflowContextDeprecationWarning: Accessing 'execution_date' from the template is deprecated and will be removed in a future version. Please use 'data_interval_start' or 'logical_date' instead.
  warnings.warn(_create_deprecation_warning(key, self._deprecation_replacements[key]))

[2024-10-16T10:05:50.851+0000] {taskinstance.py:1513} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=exchange_rate_usd_kzt_dag
AIRFLOW_CTX_TASK_ID=insert_rate
AIRFLOW_CTX_EXECUTION_DATE=2024-06-05T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2024-06-05T00:00:00+00:00
[2024-10-16T10:05:50.853+0000] {sql.py:253} INFO - Executing: INSERT INTO
    currency_exchange_rates
VALUES ('USD', 'KZT', 447.78461655, '2024-06-05')
ON CONFLICT (base, currency, date) DO
    UPDATE
        SET rate = excluded.rate;
[2024-10-16T10:05:50.874+0000] {base.py:73} INFO - Using connection ID 'postgres_default' for task execution.
[2024-10-16T10:05:51.131+0000] {base.py:73} INFO - Using connection ID 'postgres_default' for task execution.
[2024-10-16T10:05:51.143+0000] {sql.py:364} INFO - Running statement: INSERT INTO
    currency_exchange_rates
VALUES ('USD', 'KZT', 447.78461655, '2024-06-05')
ON CONFLICT (base, currency, date) DO
    UPDATE
        SET rate = excluded.rate;, parameters: None
[2024-10-16T10:05:51.147+0000] {sql.py:373} INFO - Rows affected: 1
[2024-10-16T10:05:51.163+0000] {taskinstance.py:1327} INFO - Marking task as SUCCESS. dag_id=exchange_rate_usd_kzt_dag, task_id=insert_rate, execution_date=20240605T000000, start_date=20241016T100550, end_date=20241016T100551
[2024-10-16T10:05:51.212+0000] {local_task_job.py:159} INFO - Task exited with return code 0
[2024-10-16T10:05:51.234+0000] {taskinstance.py:2582} INFO - 0 downstream tasks scheduled from follow-on schedule check
